invenio_rdm_records/templates/semantic-ui/invenio_notifications/community-submission.submitted.jinja

```diff
@@ -0,0 +1,35 @@
+{# notifications/community-submission.submitted.jinja #}
+
+{% block ctx_setup %}
+    {% set submission_request = notification.context["request"] %}
+    {%- set dummy = ctx.update({
+        "submission_request": submission_request,
+        "community": submission_request["receiver"],
+        "creator": submission_request["created_by"],
+        "record": submission_request["topic"],
+        })
+    %}
+{% endblock ctx_setup %}
+
+{%- block subject -%}
+    New record submission for your community {{ ctx["community"]["metadata"]["title"] }} submitted by {{ ctx["creator"]["username"] }}
+{%- endblock subject -%}
+
+{%- block html_body -%}
+<p>The record "{{ ctx["record"]["metadata"]["title"] }}" was submitted to your community {{ ctx["community"]["metadata"]["title"] }} by {{ ctx["creator"]["username"] }}.</p>
+
+<a href="{{ ctx["submission_request"]["links"]["self_html"] }}" class="button">Review the submission_request</a>
```

Comment:
This link is not correct. It will point to something like `<base_url>/requests/<request_uuid>`
A request, for wich user has access to, should be accessible via `<base_url>/me/requests/<request_uuid>`. But I am not sure about other implications this might have. For a community manager, it would make sense to direct them to `<base_url>/communities/<community_slug>/requests/<request_uuid>` , as this is the context where the request should be reviewed.

We could of course construct the request url per template like
```
{{ ctx["community"]["links"]["self_html] + "/requests/` + ctx["submission_request"]["id"] }}`
```
Any suggestions on this? @slint @ntarocco

Reply:
You are right. I think that this is entire point of `self_html`, and any other `_html` link in `links`: they should point to the UI that displays that object.
It is indeed complicated in case of requests, because it depends on the logged in user that will see it (e.g. invitations).

@rekt-hard @slint Does it sound crazy if we redirect, depending on the logging user? 

E.g. for invitation, the `self_html` will be `/requests/<request_uuid>` and we redirect to:
- if I am the invited user `/me/requests/<request_uuid>`
- if I am the creator (community owner/manager/curator), then `/communities/<community_slug>/requests/<request_uuid>`
This would be a way to centralize this login, and avoid to re-implement it every time we need to use the `self_html`: the `self_html` cannot know in advance what is the URL.

Reply:
This is an interesting approach. Would the responsibility for deciding the final redirect destination be on the specific request?
For current request types, can we make sure that the context for the current user will be indisputable? I.e. there is no possibility that two destinations would make sense? Because then we end up in a similar position where we have to choose one (but each will be correct at least)

Reply:
I think taking into account the user identity context for the final link at the `/requests/<request_id>` endpoint is probably much easier to encapsulate centrally in some sort of "user-friendly UI page resolver" vs. doing it when generating links... 

On the other hand, this design would call for some sort of additional logic on each request type which defines the link resolution based on the user identity... Which sounds like something that could also just be baked in the whole "link context expand logic", that [already has access to the user identity](https://github.com/inveniosoftware/invenio-records-resources/blob/58503e2a6fdc7bd110f7d3bf3a51e7ca05fd63f0/invenio_records_resources/services/base/links.py#L88).

Not sure what's best, especially without having a first go at coding it :)

Reply:
I have referenced this discussion in a new issue https://github.com/inveniosoftware/invenio-rdm-records/issues/1327 I think for now it is good to merge in the current state, since the links issue it is deeper and seems to me out of scope. to be addressed globally

Reply:
The link is still not correct though. So it should be fixed before merging. Otherwise users will get directed to a 404